### PR TITLE
Update collaboration - TO REVIEW.md

### PR DESCRIPTION
**Pull request by molnarm84**

### CODE REVIEW

File: content/concepts/collaboration.md [hunk 1]
```diff
@@ -3,3 +3,19 @@ title: "Collaboration"
 metaTitle: "sensenet - Versioning"
 metaDescription: "sensenet Versioning"
 ---
+
+In many cases developers and non-developers have to work together during a project. sensenet enables collaborative content management, keeping all your content on a single hub.
+
+# Collaboration features
+
+In sensenet it is possible to make real-time editing together with your collaborators, thanks to **Office Online Editing**.  **Versioning** feature is the management of changes to documents and other information stored in a file system or repository.
```
Comment (huszerlbence) [inline content/concepts/collaboration.md:11]:
```suggestion
In sensenet it is possible to make real-time editing together with your collaborators, thanks to **Office Online Editing**. 
**Versioning** let's you track changes of documents and other information stored in the repository.
```

File: content/concepts/collaboration.md [hunk 2]
```diff
@@ -3,3 +3,19 @@ title: "Collaboration"
...
+Using **simple approval**, contents needs to be sent for approval and can only be published when the review is finished and modification requests has been adapted.
+Created around the concept “everything is content”, in sensenet it is easy to **share** not only a single content (like a document) but a whole folder or a workspace with other users.
```
Comment (huszerlbence) [inline content/concepts/collaboration.md:13]:
```suggestion
sensenet let's you easily **share** not only a single content (like a document) but whole folders or workspace with other users.
```

File: content/concepts/collaboration.md [hunk 3]
```diff
@@ -3,3 +3,19 @@ title: "Collaboration"
 metaTitle: "sensenet - Versioning"
 metaDescription: "sensenet Versioning"
 ---
+
+In many cases developers and non-developers have to work together during a project. sensenet enables collaborative content management, keeping all your content on a single hub.
+
+# Collaboration features
+
+In sensenet it is possible to make real-time editing together with your collaborators, thanks to **Office Online Editing**.  **Versioning** feature is the management of changes to documents and other information stored in a file system or repository.
+Using **simple approval**, contents needs to be sent for approval and can only be published when the review is finished and modification requests has been adapted.
```
Comment (huszerlbence) [inline content/concepts/collaboration.md:12]:
```suggestion
With **simple approval** feature, contents needs to be sent for approval and can only be published when the review is finished and modification requests has been adapted.
```

**Review by huszerlbence — CHANGES_REQUESTED** [review]
suggested some changes, please check and approve if you agree

**Review by herflis — APPROVED** [review]

**Review by huszerlbence — APPROVED** [review]
szép munka

**Pull request merged**